### DISCUSSION TITLE
about/

### DISCUSSION THREAD

**Discussion by giscus** [hmsjy2017/blog-comment#2] [Announcements]
# about/

关于我某无聊中学生    学过一点 Python   掌握 Linux 基本操作 会写 Markdown 会写简单的网页   树莓派爱好者 Raspbian Addons 维护者  关于博客本站部署于 Github，基于 Hexo 博客系统。 文章默认使用 CC BY-NC-SA 4.0 协议进行许可，使用时请注意遵守协议。 博客最早搭建在 wordpress.com，后因限制太多尝试过自建 Wo

https://blog.iamsjy.com/about/

**Comment by hmsjy2017** [comment]
Hi

**Comment by mvocp** [comment]
博客名称：VanllaOcap's Blog
博客介绍：记录生活的屑
博客网址：https://blog.255650.xyz
博客图标：https://q.qlogo.cn/g?b=qq&nk=66870019&s=640
麻烦加友链了

**Reply by hmsjy2017** [reply]
已加

**Comment by ab8ing** [comment]
换一下头像
https://abaing.com/assets/img/favicon.jpg

**Reply by hmsjy2017** [reply]
已换

**Reply by ab8ing** [reply]
修改下信息：
本站名称：阿巴de小站
本站介绍：随便写写~
本站图标：https://abaing.top/favicon.png
本站链接：https://abaing.top

**Reply by hmsjy2017** [reply]
修改好了，图标放npm了等同步

**Comment by yuanzui-cf** [comment]
博客名称：原罪の异次元小窝
简介：(≧∇≦) b
链接：https://blog.yzcf.top
图片：https://blog.yzcf.top/upload/favicon.jpg
谢谢啦～ (≧▽≦)

**Reply by hmsjy2017** [reply]
友链已加～

**Comment by LineXic** [comment]
博客名称：LineXic书屋
简介：念念不忘，必有回响
链接：https://linexic.top
图片：https://linexic.top/logo.png
谢谢啦～ (≧▽≦)

**Reply by LineXic** [reply]
已经添加本博客友链

**Reply by hmsjy2017** [reply]
已加～

**Reply by LineXic** [reply]
申请更换链接信息

```yaml
标题:LineXic书屋
链接:https://www.linexic.top
头像:https://github.com/LineXic.png
或者:https:/www.linexic.top/avatar.webp
描述:难离难舍，想抱紧些
```

**Reply by hmsjy2017** [reply]
不好意思，现在才看到，已更新

**Comment by TXM983** [comment]
博客名称∶ MuXiaoChen🍊
博客介绍∶ 柴米油盐酱醋茶，般般都在别人家
博客网址∶ https://www.aimiliy.top/
博客图标∶ https://tuchuang.voooe.cn/images/2024/07/10/avator.webp
已添加ヾ(•ω•`)o

**Reply by hmsjy2017** [reply]
已添加

**Comment by BAGElover** [comment]
name: Tanの笔记
      link: https://sbsb.life
      vsiteshot: https://imagebed.sbsb.life/babaee448f1463935d2019ddce1b2b8.png
      avatar: https://pic.imgdb.cn/item/637dbff316f2c2beb1d0d922.jpg
      descr: 一直快乐

**Reply by hmsjy2017** [reply]
已添加

**Comment by TXM983** [comment]
已填加，但暂未在贵站看到我的链接。




MuXiaoChen
***@***.***

**Comment by kuang2714** [comment]
博主你好，茶备案项目新开张，希望得到更大的支持加入茶备案！注册得到您的专属网站备案号！希望博主多多支持！https://icp.redcha.cn

**Comment by Law-Chun-Man** [comment]
博客信息

博客名称∶ law@deb
博客介绍∶ 知識開源
博客网址∶ https://alvinlaw.srht.site/sc/, mirror: https://alvinlaw.codeberg.page/sc
博客图标∶ https://alvinlaw.srht.site/favicon.ico

感謝博主：）

**Reply by hmsjy2017** [reply]
请先添加本站的友链，谢谢！

**Comment by 1x000** [comment]
博客信息：

╰(°▽°)╯

  名字: 小胖的博客
  描述: 想法需慢摇，来个好文摇一摇
  logo: https://cdn.jsdelivr.net/gh/1x000/zdzy/tp/avatar.jpg
  链接: https://blog.moegeek.com/

感谢博主 ：）

**Reply by hmsjy2017** [reply]
已添加

**Comment by 1x000** [comment]
大佬好！前阵子脑子一热搞域名迁移，结果重定向配错了全搞砸了，SEO 惨遭归零……

实在折腾不明白了，现在决定重新用回旧域名（（

之前麻烦过大佬添加/修改过友链，麻烦大佬再帮我更新一下，另外 blog.moegeek.com 后续就不打算用了（仅做 301 重定向到旧域名直到过期），麻烦顺便移除一下这个新域名的记录~

最新博客信息如下：

Name: Xiaopang的博客
Desc: 硬件居然如此卡脚（物理）
Link: https://blog.xpdbk.com/
Avatar: https://cdn.jsdelivr.net/gh/1x000/zdzy/tp/avatar.jpg

你的友链在我这边已经更新好啦！给大佬添麻烦了，非常感谢！（）

**Reply by hmsjy2017** [reply]
没事没事，已经更新了
